playbook-website/app/javascript/components/Website/src/components/README.md

```diff
@@ -0,0 +1,5 @@
+# Components Folder
```

Comment:
My thought was that all re-usable components should live in the components folder, and they will be used in `pages/` files. So, a pages file might be ... ComponentShow.tsx or KitShow.tsx... and will use predominately components from the shared `components` folder. 

HeroImage I'd say belongs inside of this folder. But DarkModeToggle does feel odd... because it'll probably only live in one specific `layouts/{FILE}`. I'll keep marinating on where to put that one.